README.md

```diff
@@ -139,13 +129,237 @@ Response:
+         "umd_tree_cover_loss__year":2005,
+         "area__ha":3.2214358844449875,
+         "whrc_aboveground_co2_emissions__Mg":1274.5636539459229
+      },
+      {
+         "umd_tree_cover_loss__year":2006,
+         "area__ha":22.01314521037408,
+         "whrc_aboveground_co2_emissions__Mg":8167.388116836548
+      },
+      {
+         "umd_tree_cover_loss__year":2007,
+         "area__ha":0.23010256317464195,
+         "whrc_aboveground_co2_emissions__Mg":136.68091201782227
+      },
+      {
+         "umd_tree_cover_loss__year":2008,
+         "area__ha":3.7583418651858187,
+         "whrc_aboveground_co2_emissions__Mg":1579.5646076202393
+      },
+      {
+         "umd_tree_cover_loss__year":2009,
+         "area__ha":0.7670085439154732,
+         "whrc_aboveground_co2_emissions__Mg":226.95782279968262
+      },
+      {
+         "umd_tree_cover_loss__year":2010,
+         "area__ha":108.37830725525636,
+         "whrc_aboveground_co2_emissions__Mg":41855.43841171265
+      },
+      {
+         "umd_tree_cover_loss__year":2011,
+         "area__ha":12.88574353777995,
+         "whrc_aboveground_co2_emissions__Mg":4887.8897132873535
+      },
+      {
+         "umd_tree_cover_loss__year":2012,
+         "area__ha":0.07670085439154732,
+         "whrc_aboveground_co2_emissions__Mg":23.061389923095703
+      },
+      {
+         "umd_tree_cover_loss__year":2013,
+         "area__ha":1.6107179422224938,
+         "whrc_aboveground_co2_emissions__Mg":601.4241733551025
+      },
+      {
+         "umd_tree_cover_loss__year":2014,
+         "area__ha":54.30420490921551,
+         "whrc_aboveground_co2_emissions__Mg":22433.24832725525
+      },
+      {
+         "umd_tree_cover_loss__year":2015,
+         "area__ha":0.3068034175661893,
+         "whrc_aboveground_co2_emissions__Mg":119.5254955291748
+      },
+      {
+         "umd_tree_cover_loss__year":2016,
+         "area__ha":5.752564079366049,
+         "whrc_aboveground_co2_emissions__Mg":2075.9469604492188
+      },
+      {
+         "umd_tree_cover_loss__year":2017,
+         "area__ha":24.774375968469784,
+         "whrc_aboveground_co2_emissions__Mg":9848.338472366333
+      },
+      {
+         "umd_tree_cover_loss__year":2018,
+         "area__ha":29.75993150392036,
+         "whrc_aboveground_co2_emissions__Mg":11987.563570022583
+      },
+      {
+         "umd_tree_cover_loss__year":2019,
+         "area__ha":27.382205017782393,
+         "whrc_aboveground_co2_emissions__Mg":10558.882364273071
+      }
+   ]
+}
 ```
 
+### Architecture
+
+To support fast on-the-fly analytics for both small and large areas, we use a serverless AWS Lambda-based architecture. Lambdas allow us to scale up massive parallel processing very quickly, since requests usually come sporadically and may require very variable workloads.
+
+<img width="1171" alt="image" src="https://github.com/user-attachments/assets/4e359df9-c5c9-4a32-9e7a-1e6359fd03ab">
+
+We have three different lambas. 
+
```

Comment:
Also, you probably don't want to add it now, but with batch OTF, we have several more lambdas (preprocessing, list-tiled-raster-analysis, aggregation), but I can add that in a later update to this file in a short separate section.

Reply:
That'd be great, thanks. Yeah, I skipped that for now since the team is already fairly knowledgeable about that, but would good to document too, maybe with a picture or graph of the step function.